src/natcap/invest/coastal_blue_carbon/coastal_blue_carbon.py

```diff
@@ -1735,7 +1738,8 @@ def _calculate_emissions(
 
     valid_pixels = (
         (~numpy.isclose(carbon_disturbed_matrix, NODATA_FLOAT32_MIN)) &
```

Comment:
Yea, I avoided a lot of these in CBC, because it's clear from that nodata variable we won't have `nan` there and that it is a float comparison. But I don't think I was very consistent on that with other models that had a more vague defined nodata constant.

Reply:
This makes sense. I prefer this way as well, though it takes a bit of faith in our variable names.